FinSearchUnified/tests/PluginTest.php

```diff
@@ -31,4 +39,191 @@ public function testCalculateGroupkey()
         $this->assertEquals($usergroup, $decrypted);
 
     }
+
+    /**
+     * @param $number
+     * @param $isActive
+     * @return \Shopware\Models\Article\Article
+     * @throws \Shopware\Components\Api\Exception\CustomValidationException
+     * @throws \Shopware\Components\Api\Exception\ValidationException
+     */
+    private function createTestProduct($number, $isActive)
+    {
+        // create temporary test product for the export cases
+        $testArticle = array(
```

Comment:
Since SW requires PHP 5.6 as the min. version, can safely use the `[]` notation instead.